bin/export_open_targets.sh

```diff
@@ -50,8 +54,8 @@ installValidator(){
   source $venvPath/ot-validator/bin/activate
   pip install --upgrade pip==18.1
   pip install --upgrade setuptools==40.6.2
-  pip install opentargets-validator==0.4.0
-  deactivate
+  pip install opentargets-validator=="$opentargetsValidator"
+    deactivate
```

Comment:
```suggestion
deactivate
```

it seems to me that for some reason your choice of editor sometimes adds these whitespaces/tabs.